src/test/videopress/local-helpers/utils.js

```diff
@@ -123,47 +123,84 @@ export const expectShowMediaOptions = ( screen, { title, options } ) => {
 };
 
 /**
- * Generates the HTML of the VideoPress block.
+ * Generates the block settings related to privacy
+ * based on video and site privacy.
  *
  * @param {Object} options
- * @param {string} [options.guid]        VideoPress GUID.
- * @param {string} [options.title]       Title of the video.
- * @param {string} [options.description] Description of the video.
- * @param {string} [options.isPrivate]   True if the video is private.
+ * @param {string} [options.isVideoPrivate] True if the video is private. If not defined, it will use site's privacy.
+ * @param {string} [options.isSitePrivate]  True if the site is private.
```

Comment:
These appear to be inaccurate. 
```suggestion
 * @param {boolean} [options.isVideoPrivate] True if the video is private. If not defined, it will use site's privacy.
 * @param {boolean} [options.isSitePrivate]  True if the site is private.
```

Reply:
Ah, good catch, thanks for the suggestion.